types/v56/openapi.go

```diff
@@ -437,6 +437,27 @@ type DefinedInterface struct {
 	IsReadOnly bool   `json:"readonly,omitempty"` // True if the entity type cannot be modified
 }
 
+// Behavior defines a interface for a defined entity. The combination of nss+version+vendor should be unique
```

Comment:
> nss+version+vendor should be unique

I don't see these fields in this structure

Reply:
Understandable, this comment is completely wrong. Fixed.
Thanks for spotting!